### DISCUSSION TITLE
"All notes: /path/to/dir"

### DISCUSSION THREAD

**Discussion by ceskyDJ** [saber-notes/saber#819] [Ideas]
Hi @adil192! Thanks for adding support for folders. It's a breaking change for me, since I need folders a lot for my production use (currently I use Saber just for testing and maintaining of the Czech translation). You do great job, I think Saber will be the best app on the market, when it will be officially released for every day use (better than Samsung Notes, etc.). There are a few missing things, but you are on the right way!

I got one thing. You have a "All notes" text displayed every time, so when I go the the folder (and obviously don't see all notes), the "All notes" text is still displayed. I think it should be changed to something else, maybe like "Notes in /path/to/dir".

The second thing I have for you is a little problem with the path of Saber folders. I think it's really good for Unix and linux users, but not for others. It will be better to visualize folders in a more "human" form, like "path » to » dir" or something like that, maybe use some icons there).

The "All notes" isn't the best at the root folder, too. You don't see all of the notes, just some of them, because the rest is hidden in folders. So, maybe the best option will be something like (this is just a text art, I know that you can do it better):
```
Notes
⌂
```
and
```
Notes
⌂ » school » Bachelor thesis » Consultations
```

**Comment by adil192** [comment]
These are pretty good ideas. I'll convert them into issues so I don't forget!
